Extensions/Lighting/lightruntimeobject-pixi-renderer.js

```diff
@@ -75,12 +77,31 @@ gdjs.LightRuntimeObjectPixiRenderer.prototype.ensureUpToDate = function () {
 };
 
 gdjs.LightRuntimeObjectPixiRenderer.prototype.updateVertexBuffer = function () {
+  // var vertices = [this._object.x + this._radius, this._object.y + this._radius];
   this._light.shader.uniforms.center = new Float32Array([
     this._object.x + this._radius,
     this._object.y + this._radius,
   ]);
 
-  this._light.geometry
+  console.log(this.raycastTest());
+  // if (raycast.length !== 0) {
+  //   vertices.push.apply(vertices, raycast.flat(1));
+  //   this._light.geometry
+  //   .getBuffer('aVertexPosition')
+  //   .update(new Float32Array(vertices));
+
+  //   var indices = [0,1,2];
+
+  //   for(var i = 3; i < vertices.length; i+=3) {
+  //     indices[i] = 0;
+  //     indices[i + 1] = indices[i - 1];
+  //     indices[i + 2] = indices[i + 1] + 1;
+  //   }
```

Comment:
I have commented out all of this because I get empty arrays from the raycasting function. `this.raycastTest()` returns an array of vertex positions which are empty.